client/src/components/DocumentPage/Modals/VersionHistoryModal.jsx

```diff
@@ -119,53 +146,69 @@ export default function VersionHistoryModal({ documentID, onClose, quillRef }) {
             </div>
           )}
 
+          {/* --- CAROUSEL UI --- */}
```

Comment:
This should be its own component in the future, where you pass in an array of items.